lgc/patch/NggPrimShader.cpp

```diff
@@ -5491,7 +5491,7 @@ Function *NggPrimShader::createFetchCullingRegister(Module *module) {
     // regOffset = regOffset >> 2
     regOffset = m_builder->CreateLShr(regOffset, 2); // To dword offset
 
-    auto loadPtr = m_builder->CreateGEP(primShaderTablePtr, {m_builder->getInt32(0), regOffset});
+    auto loadPtr = m_builder->CreateGEP(primShaderTablePtrTy, primShaderTablePtr, {m_builder->getInt32(0), regOffset});
```

Comment:
Should this be the inner type instead of the pointer? (`ArrayType::get(m_builder->getInt32Ty(), 256)`)

Reply:
Yes. I'll fix. (Well spotted).